global.d.ts

```diff
@@ -0,0 +1,16 @@
+interface Car {
+  id: number;
+  make: string;
+  model: string;
+  year: number;
+  color: string;
+  mileage: number;
+  price: number;
+  fuelType: "Gasoline" | "Electric";
```

Comment:
It would be better to create enum type for values like this:
`enum FuelType {Gasoline, Electric}; `
then just use the enum here:
```suggestion
  fuelType: FuelType;
```

Same for transmission.